src/main/java/com/google/graphgeckos/dashboard/api/components/BuildInfo.java

```diff
@@ -0,0 +1,53 @@
+package com.google.graphgeckos.dashboard.api.components;
+
+import java.util.ArrayList;
+import java.util.List;
+
+// Represents the response json with the aggregated buildbot data.
+public class BuildInfo {
+
+  // git commit hash of the tested changes commit
+  private final String commitHash;
+  /*
+   the relative time (time since the application was started),
+   allows to sort received build info in the descending order (latest goes first)
+   */
```

Comment:
nit: it's a bit unwieldy to mix `//`-style and `/*`-style comments in this section. I'd stick to one style, e.g., turn this section into `//`-style.